functions/gdl/game_controller.py

```diff
@@ -38,11 +46,24 @@
 @API.route('/', strict_slashes=False)
 class Games(Resource):
 
-    @API.doc('List of available games', params={'language': 'Optional BCP47 language code to filter results'})
-    @API.marshal_list_with(game, skip_none=True)
+    @API.doc('List of available games', params={'language': 'Optional BCP47 language code to filter results', 'page': 'Return results for this page', 'page-size': 'Return this many results per page.' })
+    @API.marshal_with(game_list, skip_none=True)
     def get(self):
         lang = flask.request.args.get('language')
-        return game_repository.all_with_language(lang) if lang else game_repository.all()
+        page = flask.request.args.get('page')
+        page_size = flask.request.args.get('page-size')
```

Comment:
En ting jeg lurer på her - er det ikke bedre å bare gi ut X spill over Y sider som default hvis man ikke oppgir noe argument for `page` og `page-size`?

Reply:
Hvis jeg forstod deg riktig, så har jeg satt default om mann ikke oppgir `page` og `page-size` til å vise 10 spill av side 1. Akkurat som bok-apiet. Var det det du mente?

Reply:
Ja, nvm, tenkte feil. Men du fikk ikke satt default values i funksjonsdeklarasjonen?

Reply:
Som vi diskuterte; så var det slikt at vi ikke fikk satt default verdi pga de overskriver default values i funksjonsdeklarasjonene.

Reply:
Hele denne blokken med å hente ut parametrene `page` og `pageSize` tror jeg kan gjøres litt mer sofistikert.
Som @vikre sier, så savner jeg default-verdier. Hva med å prøve noe ala dette: `page = request.args.get('page', default = 1, type = int)`